src/datetimepicker.ios.js

```diff
@@ -89,6 +90,7 @@ export default class Picker extends React.Component<IOSNativeProps> {
         minuteInterval={minuteInterval}
         timeZoneOffsetInMinutes={timeZoneOffsetInMinutes}
         onChange={this._onChange}
+        textColor={textColor}
```

Comment:
afaik we should first call `processColor` on this, for example see https://github.com/facebook/react-native/blob/ce0edca620d49597aa73c71a68b70efa170bc828/Libraries/Share/Share.js#L110

please also include the invariant check for color correctness, thank you

Reply:
I can't get it to work with processColor though. Any ideas?

Reply:
I'll investigate again tomorrow 😪

Reply:
@sbycrosz sorry, if that does not work then I was probably wrong. The conversion is probably done automatically since you specified it's a `UIColor` in `RCT_CUSTOM_VIEW_PROPERTY(textColor, UIColor, RNDateTimePicker)`

Reply:
I managed to get this to work by using NSNumber instead of UIColor (as seen in https://github.com/facebook/react-native/issues/3014)